File: example-app/src/main/java/io/lsdconsulting/intercceptors/example/FishApp.java
package io.lsdconsulting.intercceptors.example;

import io.lsdconsulting.intercceptors.example.config.AppConfiguration;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Import;

@Import(AppConfiguration.class)
@SpringBootApplication
public class FishApp {

    public static void main(String[] args) {
        SpringApplication.run(FishApp.class, args);
    }
}

File: example-app/src/main/java/io/lsdconsulting/intercceptors/example/FishApp.kt
package io.lsdconsulting.intercceptors.example

import io.lsdconsulting.intercceptors.example.config.AppConfiguration
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.Import

@Import(AppConfiguration::class)
@SpringBootApplication
open class FishApp

fun main(args: Array<String>) {
    SpringApplication.run(FishApp::class.java, *args)
}
